01_basic_commands/020_search_files/search_files.md

```diff
@@ -1 +1,139 @@
-Search for files
+# Search For Files
+
+Multiple approaches for searching for files are offered. If you are searching for a file by its file name `locate` is the choice, `find` is used to search for files by its attributes, `grep` allows searching for files via their content.
```

Comment:
Why are you mentioning grep? I did not find any explanation of the grep in the following sections.